linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/template/router_client.template

```diff
@@ -2,27 +2,58 @@
   {{client}}
 </div>
 
-<div class="client-metrics">
-  <dl>
-  {{#each metrics}}
+<div class="client-metrics col-md-4">
+  <div class="row">
     <div class="col-md-2">
-      <dt>
-        {{description}}
-      </dt>
+      <dt>Requests</dt>
+      <dd>{{requests}}</dd>
+    </div>
+    <div class="col-md-2">
+      <dt>Connections</dt>
+      <dd>{{connections}}</dd>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-2">
+      <dt>Successes</dt>
+      <dd>
+        {{#if successRate}}
+          {{successRate}}
+        {{else}}
+          {{success}}
+        {{/if}}
+      </dd>
+    </div>
+    <div class="col-md-2">
+      <dt>Failures</dt>
       <dd>
-        <span class="metric-medium">
-          {{#if rate}}
-            {{rate}}
-          {{else}}
-            {{#if value}}
-              {{value}}
-            {{else}}
-              0
-            {{/if}}
-          {{/if}}
-        </span>
+        {{#if failures}}
+          {{failures}}
+        {{else}}
+          0
+        {{/if}}
       </dd>
     </div>
-  {{/each}}
-  </dl>
+  </div>
+</div>
+
+<div class="col-md-2">
+  <div>Latencies</div>
```

Comment:
Super minor -- I think it might make sense to bold the Latencies heading, so that it's more similar to the Requests, Failures, Connections, etc. heading from above.  Right now it looks like:

![screen shot 2016-04-05 at 3 36 03 pm](https://cloud.githubusercontent.com/assets/9226/14300280/4d96c052-fb44-11e5-984a-d1f07c02168f.png)